129 style main sidebar separation bar

NatanAmado opened the pull request.

ovirandunu commented:
please stop copy pasting entire files with chatgpt.

@noahlabear all functionality needs to be retested for any changed components because some copy pastes seem to have occured. (Can see meaninglessly removed comments and refactored code)

NatanAmado commented:
> please stop copy pasting entire files with chatgpt.
> 
> @noahlabear all functionality needs to be retested for any changed components because some copy pastes seem to have occured. (Can see meaninglessly removed comments and refactored code)

I know exactly was changes were made. In this case, the only thing that I modified using AI was the layout and just for the sake of quickly styling the separation bar, but all the functionality is unchanged.
